cmd/collector/app/grpc_handler.go

```diff
@@ -43,7 +43,10 @@ func (g *GRPCHandler) PostSpans(ctx context.Context, r *api_v2.PostSpansRequest)
 			span.Process = r.Batch.Process
 		}
 	}
-	_, err := g.spanProcessor.ProcessSpans(r.GetBatch().Spans, JaegerFormatType)
+	_, err := g.spanProcessor.ProcessSpans(r.GetBatch().Spans, ProcessSpansOptions{
+		InboundTransport: "grpc", // TODO do we have a constant?
```

Comment:
Could we create constants if they don't exist?

Reply:
Jude is doing it in the next PR. Right now these strings are not used anywhere.

Reply:
Yes, I have the followup story to address that.